packages/api/src/routers/api/team.ts

```diff
@@ -158,4 +158,48 @@ router.get('/tags', async (req, res, next) => {
   }
 });
 
+router.delete('/users/:userEmail', async (req, res, next) => {
+  try {
+    const teamId = req.user?.team;
+    const userEmail = req.params.userEmail;
```

Comment:
my preference here is that we're able to remove users by `id` instead of email (invites as well, but that isn't as big of a priority).

Reply:
second this. should use user id instead

Reply:
Thanks for the review, changes made